b/byte-buddy-byte-buddy/byte-buddy-byte-buddy_ubi_9.3.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e   
+# ----------------------------------------------------------------------------
+#
+# Package       : byte-buddy-byte-buddy
+# Version       : byte-buddy-1.12.23
+# Source repo   : https://github.com/raphw/byte-buddy
+# Tested on     : UBI: 9.3
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : kotla santhosh<kotla.santhosh@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+REPO_NAME="byte-buddy"
+PACKAGE_NAME=byte-buddy
```

Comment:
Dont need double variables. Can keep one variable only since they have the same value.